cmd/kubectl-gadget/seccompadvisor.go

```diff
@@ -65,9 +65,9 @@ func init() {
 		"output-mode", "m",
 		"terminal",
 		"The trace output mode, possibles values are terminal and seccomp-profile.")
-	seccompAdvisorStartCmd.PersistentFlags().StringVar(&seccompProfileName,
-		"seccomp-profile-name", "",
-		"Name of the seccomp profile to be created when using --output-mode=seccomp-profile.\nNamespace can be specified by using namespace/profile-name.")
+	seccompAdvisorStartCmd.PersistentFlags().StringVar(&profilePrefixName,
+		"profile-prefix-name", "",
```

Comment:
I think I was the one suggesting `profile-prefix-name`. I think it's not the best name for this argument. Perhaps `profile-name-prefix` is better?

Reply:
Not sure. What do you think? @alban @marga-kinvolk @eiffel-fl

Reply:
`profile-name-prefix` seems better regarding grammar but `profile-prefix-name` sounds better when pronounced :rofl:.
I would say we should go with the one which respects the grammar.

Reply:
What about `profile-prefix`?

Reply:
I used `profile-prefix`.